scripts/globals/items/pinch_of_fanatics_powder.lua

```diff
@@ -0,0 +1,43 @@
+-----------------------------------------
+-- ID: 5847
+-- Item: pinch_of_fanatics_powder
+-- Item Effect: When applied, grants UDMGPHYS -10000 to party members in range for 60s
+-----------------------------------------
+require("scripts/globals/status")
+require("scripts/globals/msg")
+-----------------------------------------
+local item_object = {}
+
+item_object.onItemCheck = function(target)
+    return 0
+end
+
+item_object.onItemUse = function(target)
+    local shieldtype = xi.effect.PHYSICAL_SHIELD
+    local duration = 60
+    local power = 1
+    local wsmitigation = 1
+
+    local function addShield(target, power, duration)
```

Comment:
target is already defined in the main function.  My recommendation would be to not include this function definition within an existing function, but rather outside of it.